dll/win32/msgina/msgina.rc

```diff
@@ -60,7 +60,8 @@ IDB_IMAGE_STRIP BITMAP "resources/image_strip.bmp"
 #ifdef LANGUAGE_JA_JP
     #include "lang/ja-JP.rc"
 #endif
-#ifdef LANGUAGE_NB_NO
+//#ifdef LANGUAGE_NB_NO
+#ifdef LANGUAGE_NN_NO
```

Comment:
delete the commented line, and that diff will vanish entirely!

Reply:
Can you just stop reviewing this draft PR? For your msgina.rc "doesn't make any change" statement, you could just have looked at the first "temp" commit!